src/urbit/app/chess.hoon

```diff
@@ -786,13 +789,17 @@
     [%poke %challenge %send ~]
       ?+  -.sign  (on-agent:default wire sign)
         %poke-ack
-          ::  if opponent has acked our challenge, do nothing
+          ::  if opponent has acked our challenge, add it to the frontned
           ?~  p.sign
+          ::  
             :-  :~  :*  %give  %fact   ~[/challenges]
-                        %chess-update  !>([%challenge-sent src.bowl (~(get by challenges-sent) src.bowl)])
+                        %chess-update  !>([%challenge-sent src.bowl q:(rear ~(tap by challenges-sent))])
```

Comment:
Left a comment before your PR a few minutes ago, but editing it now for your latest commit: this should just be `got`